BuildResidentialHPXML/tests/base-foundation-ambient.osw

```diff
@@ -130,7 +130,7 @@
         "geometry_num_floors_above_grade": 1,
         "geometry_num_occupants": "3",
         "geometry_orientation": 180.0,
-        "geometry_rim_joist_height": 0,
+        "geometry_rim_joist_height": 9.25,
```

Comment:
An ambient foundation shouldn't have rim joists. I suggest reverting this and changing `rim_joist_assembly_r` to zero instead. (Or should we be omitting these rim joist arguments for OSWs where rim joists don't apply?)

Reply:
I will omit.